eradiate/radprops/tests/test_rad_profile.py

```diff
@@ -85,9 +81,20 @@ def test_us76_approx_rad_profile(
         assert isinstance(x, ureg.Quantity)
 
 
+def test_us76_approx_rad_profile_ckd_sigma_s(mode_ckd):
+    """
+    Scattering coefficient evaluation methods return pint.Quantity object.
+    """
+    p = US76ApproxRadProfile()
+
+    spectral_ctx = CKDSpectralContext.new()
```

Comment:
Here you can be lazy and just use
```suggestion
    spectral_ctx = SpectralContext.new()
```